medusa/post_processor.py

```diff
@@ -153,29 +153,45 @@ def _check_for_existing_file(self, existing_file):
             return PostProcessor.DOESNT_EXIST
 
     @staticmethod
-    def _search_files(path, pattern='*', subfolders=None, base_name_only=None, sort=True):
+    def _search_files(path, pattern='*', subfolders=None, base_name_only=None, sort=None):
         """
         Search for files in a given path.
 
-        :param path: path to file or folder (NOTE: folder paths must end with slashes)
+        :param path: path to file or folder (folder paths must end with slashes)
         :type path: text_type
         :param pattern: pattern used to match the files
         :type pattern: text_type
         :param subfolders: search for files in subfolders
         :type subfolders: bool
-        :param base_name_only: only match files with the same file name
+        :param base_name_only: only match files with the same name
         :type base_name_only: bool
         :param sort: return files sorted by size
         :type sort: bool
```

Comment:
I usually use None to make clear it's an optional argument, but it doesn't really make a difference.